frontend/src/metabase/query_builder/actions.js

```diff
@@ -1162,10 +1153,14 @@ export const archiveQuestion = createThunkAction(ARCHIVE_QUESTION, (questionId,
         }
         let response = await CardApi.update(card)
 
-        dispatch(addUndo(createUndo(
-            archived ? "archived" : "unarchived",
-            archiveQuestion(card.id, !archived)
-        )));
+        const type = archived ? "archived" : "unarchived"
+
+        dispatch(addUndo(createUndo({
+            type,
+            // eslint-disable-next-line react/display-name
+            message: () => <div> { "Question  was " + type + "."} </div>,
```

Comment:
Any reason you're not using a template literal here? `Question was ${type}.`

Reply:
where's the i18n!